exchanges/gateio/gateio_websocket.go

```diff
@@ -764,6 +733,146 @@ func singleSymbolChannel(name string) bool {
 	return false
 }
 
+// ValidateSubscriptions implements subscription.SubscriptionsValidator interface
```

Comment:
🐒  Since this is doing really just one thing, feels like we can move the comment up for clarity
```suggestion
// ValidateSubscriptions implements subscription.SubscriptionsValidator interface to ensure that conflicting orderbooks aren't subscribed to
```